src/main/webapp/css/main.css

```diff
@@ -152,7 +152,13 @@ body {
 /* chat page messages. */
 
 #messagesContainer {
-    padding: 10px;
+  padding: 10px;
+}
+
+#userResult{
+  max-height: 300px;
+  overflow-y:scroll;
```

Comment:
Let's make this `overflow-y: auto` instead. `scroll` makes the scrollbar always visible. It's ok if you're doing that on purpose though.

Reply:
This is a more general comment, maybe we should do that all over.

Reply:
It's visible only when you scroll. It quickly disappears when you stop but I took the screen shot while scrolling just to make it clear in the photos. But if auto is preferred, I can do that.

Reply:
That's because you're on a mac. Windows and Linux machines will show a fat ugly scrollbar all the time. 🙀 
As I said though, it's fine if this is the intended behavior, just raising the point.

Reply:
Oh 😅. In that case, yeah I think auto is better. Thanks!